apps/frontend/packages.d.ts

```diff
@@ -27,3 +27,10 @@ declare module "eslint-plugin-jsx-a11y" {
 		configs,
 	};
 }
+
+declare module "*.svg" {
+	import React from "react";
+
+	const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;
+	export default ReactComponent;
+}
```

Comment:
What if we add `/// <reference types="vite-plugin-svgr/client" />` to vite-env.d.ts, would that work?

Reply:
If I add this to  vite-env.d.ts, and remove *.svg declaration from packages.d.ts it resolves svg imports to strings

Reply:
According to docs this should be added. Add this and keep the declaration in .d.ts file